portal-impl/src/com/liferay/portlet/social/service/impl/SocialActivityCounterLocalServiceImpl.java

```diff
@@ -289,7 +296,15 @@ public void addActivityCounters(SocialActivity activity)
 				activityDefinition.getActivityCounterDefinition(
 					activityCounter.getName());
 
-			if (checkActivityLimit(user, activity, activityCounterDefinition)) {
+			if (activity.getType() ==
```

Comment:
Same as before. I don't think we need another special condition here for REVOKE. We should have the exact same logic as before but in liferay-social.xml we should have a negative number instead of a positive one so when the activity is counted it will substract to the already existing value. We shouldn't need any decrementActivityCounter neither.
Is there anything I'm missing?
Thanks!